tests/storage/hpp/test_hpp_node_placement.py

```diff
@@ -65,6 +65,10 @@ def test_create_dv_on_right_node_with_node_placement(
     assert cirros_vm_for_node_placement_tests.vmi.node.name == worker_node1.name
 
 
+@pytest.mark.xfail(
+    reason=f"{QUARANTINED}: fails since 4.19; CNV-66721",
+    run=False,
+)
```

Comment:
Please also create a card for `test_upload_https_scratch_space_delete_pvc`
And quarantine the test